### ISSUE TITLE
Scrabble board recognition API

### ISSUE THREAD

**Issue by Macok** [scrabscrap/scrabble-scraper-v2#180]
### Summary

Hello,

I didn't find any contact info so I'm messaging you here.

I thought maybe you'd be interested to use my API that uses Neural Networks for recognizing Scrabble boards from photos, and therefore achieves better accuracy than OpenCV:
https://scrabblecam.com/api

The API is free to use and I'm planning to keep it this way as long as the traffic is not too high.
It doesn't support German letters at the moment, but maybe you are planning to release 
English version too? (Or French/Italian).

Best,
Maciej

### Basic Example

.

**Comment by scrabscrap** [comment]
Hello Maciej,

Thank you for reaching out and sharing your API with me. Your project sounds fantastic, and I appreciate the offer to use your Neural Network-based solution for recognizing Scrabble boards.

I conducted some successful tests with your API and was impressed with the results. However, the ScrabScrap project is designed with a couple of specific requirements:
a) It needs to run offline.
b) It aims to avoid using services that might incur licensing fees or additional costs.

If your AI model is freely available and if there is a possibility of creating a TensorFlow version that could run on a Raspberry Pi, I would be very interested in exploring its integration.

Best regards,
Rainer

**Comment by scrabscrap** [comment]
If you still have questions, feel free to open the ticket again.
